types.js

```diff
@@ -143,6 +144,26 @@ export type SJFillImage = {|
   },
 |};
 
+// '{0, 1}'
+type PointString = string;
+
+type SJPoint = {|
```

Comment:
I think it would be clearer to call this `SJCurvePoint` to match the internal type